### DISCUSSION TITLE
Area information of the cells

### DISCUSSION THREAD

**Discussion by hhy2022** [idaholab/moose#21556] [Q&A General] [answered]
Hi, 

I would like to know the area information of each cells. I would used this to couple with some variable. Is there any kernel I can used for those cell areas?

Thank you so much.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You may save the volume information in an auxvariable using a VolumeAux. In 2D it gives the area
https://mooseframework.inl.gov/source/auxkernels/VolumeAux.html

Guillaume
